services/state/state.js

```diff
@@ -188,18 +188,19 @@ export function getAllLabelsForSides({ A, B, C }) {
 	return { focusGroup, topic, medium };
 }
 
+function assignRandomSide(cube) {
+	const random = Math.floor(Math.random() * 6) + 1;
+	setDiceSide(`${cube}${random}`);
+}
+
 /**
  * Returns all labels as array for the current dice sides
  * @returns {{focusGroup: string[], topic: string[], medium: string[]}}
  */
 export function getAllLabelsForCurrentSides() {
-	if (dices.A.side === null || dices.B.side === null || dices.C.side === null) {
-		return {
-			focusGroup: dices.A.side,
-			topic: dices.B.side,
-			medium: dices.C.side,
-		};
-	}
+	dices.A.side === null && assignRandomSide("A");
+	dices.B.side === null && assignRandomSide("B");
+	dices.C.side === null && assignRandomSide("C");
```

Comment:
I'd suggest to not assign sides. Because then it makes it harder to see if a dice is working properly or not. let's just use/return random values if a dice has no side.

Reply:
I first did that, but we need to assign the sides since later in the pick-idea file the cube sides are requested again.